core/src/main/java/org/apache/carbondata/core/metadata/schema/table/TableSchemaBuilder.java

```diff
@@ -108,6 +122,20 @@ public TableSchema build() {
       schema.setTableProperties(property);
     }
 
+    if (isLocalDictionaryEnabled) {
+      property.put(CarbonCommonConstants.LOCAL_DICTIONARY_ENABLE,
+          String.valueOf(isLocalDictionaryEnabled));
+      String localdictionaryThreshold =
+          localDictionaryThreshold.equalsIgnoreCase("0") ? null : localDictionaryThreshold;
```

Comment:
Need to set default value instead of null ?

Reply:
Done